mcs_benchmark_data/models/benchmark_antecedent.py

```diff
@@ -23,7 +26,14 @@ def to_rdf( self, *, graph: Graph) -> Resource:
         for element in self.elements:
             if isinstance(element,BenchmarkContext):
                 resource.add(MCS.benchmarkContext, element)
-            element.to_rdf(graph)
+            elif isinstance(element, BenchmarkQuestion):

```

Comment:
Likely the better way is to have the various elements point to this e.g., BenchmarkContext has a benchmark_antecedent_uri: URIRef.